config/console.php

```diff
@@ -3,10 +3,15 @@
 use Psr\Container\ContainerInterface;
 use Symfony\Component\Console\CommandLoader\ContainerCommandLoader;
 use Yiisoft\Yii\Console\Application;
+use Yiisoft\Yii\Console\SymfonyEventDispatcher;
 
 return [
     Application::class => function (ContainerInterface $container) use ($params) {
         $app = new Application();
+
+        $dispatcher = $container->get(SymfonyEventDispatcher::class);
```

Comment:
Does SymfonyEventDispatcher support event inheritance as our dispatcher does?

Reply:
This dispatcher is just a wrapper used for symfony console events